### DISCUSSION TITLE
500 Server Error since yesterday

### DISCUSSION THREAD

**Discussion by jmun304** [community/community#130797] [GitHub Education]
### Select Topic Area

Question

### Body

Hello,
I cannot accept new assignments, I am getting a 500 error after series of reaccepting them....

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @jmun304 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what your question entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
